console/web3ext/web3ext.go

```diff
@@ -716,6 +716,17 @@ web3._extend({
 			params: 2,
 			inputFormatter: [web3._extend.formatters.inputAddressFormatter, web3._extend.formatters.inputDefaultBlockNumberFormatter],
 		}),
+		new web3._extend.Method({
+			name: 'getHeaderByNumber',
+			call: 'klay_getHeaderByNumber',
+			params: 1,
+			inputFormatter: [web3._extend.formatters.inputBlockNumberFormatter]
+		}),
+		new web3._extend.Method({
+			name: 'getHeaderByHash',
+			call: 'klay_getHeaderByHash',
+			params: 1
+		}),
```

Comment:
`~NumberOrHash` API methods are implemented, but they are not introduced in web3ext. Why don't you add them?